src/scss/components/_breadcrumbs.scss

```diff
@@ -1,132 +1,137 @@
-/**
- * Components > Breadcrumb
- * =================================
- * Used for website navigation breadcrumbs
- *
- * Documentation:
- * https://fozzie.just-eat.com/styleguide/ui-components/breadcrumbs
- */
-
-$breadcrumb-font-weight: $font-weight-bold;
-$breadcrumb--compact-background: rgba($black, 0.2);
-
-.c-breadcrumb {
-    width: 100%;
-    line-height: 2.5;
-    @include media('>=mid') {
-        @include font-size(body-l, false);
-    }
-}
-
-.c-breadcrumb-item {
-    float: left;
-    color: $grey--dark;
-    svg {
-        fill: $black;
+@mixin breadcrumbs() {
+    /**
+    * Components > Breadcrumb
+    * =================================
+    * Used for website navigation breadcrumbs
+    *
+    * This is an optional component within Fozzie.
+    * If you'd like to use it in your project you can include it by adding `@include breadcrumbs();` into your SCSS dependencies file.
+    *
+    * Documentation:
+    * https://fozzie.just-eat.com/styleguide/ui-components/breadcrumbs
+    */
+
+    $breadcrumb-font-weight: $font-weight-bold;
+    $breadcrumb--compact-background: rgba($black, 0.2);
+
+    .c-breadcrumb {
+        width: 100%;
+        line-height: 2.5;
+        @include media('>=mid') {
+            @include font-size(body-l, false);
+        }
     }
-}
-
-.c-breadcrumb-item--home {
-    padding-left: 0;
-}
-
-.c-breadcrumb-item-link {
-    text-decoration: none;
-    color: $grey--darkest;
-    font-weight: $breadcrumb-font-weight;
 
-    &:hover {
-        text-decoration: underline;
+    .c-breadcrumb-item {
+        float: left;
         color: $grey--dark;
+        svg {
+            fill: $black;
+        }
     }
-}
 
-.c-breadcrumb-item-icon {
-    margin: 0 spacing(base);
-}
+    .c-breadcrumb-item--home {
+        padding-left: 0;
+    }
 
-.c-breadcrumb--transparent {
-    background-color: transparent;
+    .c-breadcrumb-item-link {
+        text-decoration: none;
+        color: $grey--darkest;
+        font-weight: $breadcrumb-font-weight;
 
-    .c-breadcrumb-item {
-        color: $white;
-        svg {
-            fill: $white;
+        &:hover {
```

Comment:
`&:focus`?

Reply:
If it's ok, I'll leave this as is (as I was only changing the indentation in these files). I'll test it in a subsequent PR though to see if it needs the focus state.